main.go

```diff
@@ -45,7 +45,13 @@ func init() {
 // New setups the configuration assuming various parameters have been setup in the AWS account
 func New() (h handler, err error) {
 
-	cfg, err := external.LoadDefaultAWSConfig(external.WithSharedConfigProfile("uneet-dev"))
+// THIS IS WRONG: we should not use "ins-dev" here but the correct value based on
+//  - Installation ID (AWS parameter `INSTALLATION_ID`)
+//	- Stage (AWS parameter `STAGE`) 
+// The value for "ins-dev" is also declared in Travis Setting
+// it is then passed on as the variable `TRAVIS_PROFILE`
+	cfg, err := external.LoadDefaultAWSConfig(external.WithSharedConfigProfile("ins-dev"))
```

Comment:
The way this line works, is that it uses "ins-dev" locally, else it uses the current execution environment.

So imo it should be set to the environment you develop in. E.g. uneet-dev in this case

Reply:
> So imo it should be set to the environment you develop in. E.g. uneet-dev in this case

The problem with this approach is that this means the code will not be usable "as is" in the Unee-T INS fork <-- this will make things more obscure as the people in charge on the Unee-T INS code will have to know that this repo has to be forked and modified in order for their installation to work as intended.

Much better to use a variable that is already declared in the AWS parameter store and in the Travis settings instead.

See https://github.com/unee-t/apienroll/issues/4 for more details